src/components/text/Text.tsx

```diff
@@ -1,15 +1,21 @@
-import { ParagraphStylesMap } from "./Text.constans.tsx";
+import { ParagraphStylesMap, TextColorsMap } from "./Text.constans.tsx";
 import { TextStyled } from "./Text.styles.ts";
 import { TextTypes } from "./Text.types.tsx";
 
-const Text = ({ variant = "small", children }: TextTypes) => {
+const Text = ({
+  variant = "small",
+  alertType = "default",
```

Comment:
the idea behind this prop is very specific. It suggests that the Text is mainly used to alert about something like a toast or sth similar. 

Since the only thing this does is set a color - maybe you can simply pass `color` prop instead?  You can limit the colors with Typescript if you want to. 

Up to you :)

Reply:
I agree. I had my doubts with calling it alertType